test/models.json

```diff
@@ -2157,6 +2157,38 @@
     "format":   "Hickle Weights",
     "link":     "https://github.com/lutzroeder/netron/issues/998"
   },
+  {
+    "type":     "kann",
+    "target":   "GoogLeNet-ILSVRC2014",
+    "source":   "https://github.com/kalray/netron-test-models/raw/main/GoogLeNet-ILSVRC2014-fp16-5c.kann",
+    "format":   "KaNN"
+  },
+  {
+    "type":     "kann",
+    "target":   "MobileNet-v2-ILSVRC2012",
+    "source":   "https://github.com/kalray/netron-test-models/raw/main/MobileNet_v2-ILSVRC2012-5c.kann",
+    "format":   "KaNN",
+    "tags":     "quantization"
+  },
+  {
+    "type":     "kann",
+    "target":   "ResNet50-v1.5-MLPerf",
+    "source":   "https://github.com/kalray/netron-test-models/raw/main/ResNet50v15_MLPerf_quantized-fp16-5c.kann",
+    "format":   "KaNN",
+    "tags":     "quantization"
+  },
+  {
+    "type":     "kann",
+    "target":   "SqueezeNet-v1.0-ILSVRC2012",
+    "source":   "https://github.com/kalray/netron-test-models/raw/main/SqueezeNetV1_0-ILSVRC2012-fp16-5c.kann",
```

Comment:
Can these be moved to an official model zoo repo that is being maintained.

Reply:
Our model zoo repo isn't public yet, but this repository is completely maintained and will be merged into the official zoo repo once it becomes public.

Reply:
Please front load this work to avoid churn and make it clear that these are official supported files not directly related to Netron.

Reply:
In progress, but this will result in the models not being accessible for now